compiler/src/codegen/compcore.re

```diff
@@ -332,7 +333,7 @@ let runtime_function_imports =
         mimp_type:
           [@implicit_arity]
           MFuncImport(
-            BatList.init(Runtime_errors.max_arity + 1, _ => I32Type),
+            List.init(Runtime_errors.max_arity + 1, _ => I32Type),
```

Comment:
😍 Very happy to see this is in the stdlib now

Reply:
I enjoyed my usage in `String_utils.explode`, very handy method!